models/aeif_cond_beta_multisynapse.h

```diff
@@ -64,6 +64,48 @@
  reversal potentials are supplied by the array "E_rev". The port numbers
  are automatically assigned in the range from 1 to n_receptors.
  During connection, the ports are selected with the property "receptor_type".
+The membrane potential is given by the following differential equation:
+C dV/dt= -g_L(V-E_L)+g_L*Delta_T*exp((V-V_T)/Delta_T)-g_e(t)(V-E_e)
+                                                     -g_i(t)(V-E_i)-w +I_e
```

Comment:
Ok, sorry, one last comment: since it's multisynapse, here, you should probably detail:

    C dV/dt= -g_L(V-E_L) + g_L*Delta_T*exp((V-V_T)/Delta_T) + I_syn_tot(V, t) - w + I_e
    
    where:
    I_syn_tot(V,t) = \sum_i g_i(t) (V - E_{rev,i})

    and the synapse i is excitatory or inhibitory depending on the value of E_{rev,i}.